mentorship ios/Constants/DesignConstants.swift

```diff
@@ -70,16 +70,26 @@ struct DesignConstants {
         static let accepted = Color.green
         static let rejected = Color.pink
         static let cancelled = Color.gray
-        
+
         static let indigoUIColor = UIColor.systemIndigo
         static let secondaryUIBackground = UIColor.secondarySystemGroupedBackground
     }
 
     struct Fonts {
         static let userError = Font.subheadline
-        
+
         struct Size {
             static let insetListIcon: CGFloat = 20
+            static let navBarIcon: CGFloat = 16
+        }
+    }
+    
+    struct DateFormat {
+        static var mediumDate: DateFormatter {
+            let formatter = DateFormatter()
+            formatter.dateStyle = .medium
+            formatter.timeStyle = .none
+            return formatter
```

Comment:
DateFormatter is not a design thing, we can put it in utility folder. Don't worry this can be done during project refactor period next week.

Reply:
Oh okay. I thought since it is being used to control the way the data is presented, it can considered a design decision.